core/src/main/java/binnie/core/util/EmptyHelper.java

```diff
@@ -0,0 +1,7 @@
+package binnie.core.util;
+
+public final class EmptyHelper {
+    private EmptyHelper() {}
+
+    public static final String EMPTY_STRING = "";
```

Comment:
I think this is probably overkill, I'd be surprised if the JVM does not have some kind of optimization for the empty string.

Reply:
Hm. Probably you are right.
But now I used StringUtils.Empty (as you can see in #504)

Reply:
Ok, sounds good.